### ISSUE TITLE
Dyson App - Digital Assistant [boldchat.com]

### ISSUE THREAD

**Issue by hagezi** [hagezi/dns-blocklists#242]
Hello,

The dyson app's digital assistant gives an error when connecting. When I check it, hagezi is blocked by multi pro++. I think this is a false hindrance.

![9C0138D1-5E1F-441B-A38D-C08E428E2DEE](https://user-images.githubusercontent.com/30311445/214338473-8b24728e-6a01-41f0-a897-7a03377a4770.jpeg)
![58235F85-1A42-43FF-A773-58DEEB25FAC6](https://user-images.githubusercontent.com/30311445/214338496-e0a69e57-e333-44f4-96ab-dd4f88a83f63.jpeg)

_Originally posted by @azafrak in https://github.com/hagezi/dns-blocklists/issues/239#issuecomment-1402158237_

**Comment by hagezi** [comment]
@azafrak Thanks, removed.

ping @developerdan @migueldemoura

**Comment by hagezi** [comment]
@azafrak updated list is live in NextDNS, please check if it works now.

**Comment by azafrak** [comment]
> @azafrak updated list is live in NextDNS, please check if it works now.

I checked it, it was removed. But then I noticed that this site was also blocked mcpwgrxj4n94hv1pvdyqw0zwdxyq.device.marketingcloudapis.com

But these sites actually collect data. As far as I understand, when the developer who wrote the dyson application could not access these sites, the application did not open the chat section.

So in summary, I don't know how accurate it would be to remove these addresses from the list.

**Comment by hagezi** [comment]
@azafrak I had a look at the API documentation, the API can be used for tracking and non-tracking operations, like push, messaging and so on.
See: https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/apis-overview.html

As it is, it is needed for chat, so I only whitelisted the specific needed subdomain (API endpoint) for the Dyson app.

**Comment by hagezi** [comment]
@azafrak updated list is live in NextDNS, please check if it works now.

**Comment by azafrak** [comment]
I understand, thank you for your interest. Then I would like to specify another address.

Photo comments do not open in the Trendyol application. When you add this address to the whitelist, the problem improves.

Hagezi multi pro++ blocks.

![D35EE9CB-97D7-48C3-91D5-BFE7C610A124](https://user-images.githubusercontent.com/30311445/214381981-824e5269-aea5-4d26-a657-2a3fee91ea27.png)
![A0DC41A1-0184-4C5C-A001-F00962395C14](https://user-images.githubusercontent.com/30311445/214382019-106db8b3-4ba5-4a5e-8613-4907164d49df.jpeg)

**Comment by hagezi** [comment]
Thanks, I'll take a look as well, I've opened a new issue for this so the topics don't get mixed up. Is the Digital Assistant working now?

**Comment by hagezi** [comment]
I checked it in the app, the Digital Assistant is now working.

**Comment by DeveloperDan** [comment]
Hi @hagezi. You're pinging the wrong person. I'm not involved in this project.
